pooranjoyb/popShop
# Fixed bugs

Fixed 2 bugs.
1. Now it is mandatory to add a size before adding an item to the cart.
2. After the checkout, all items are cleared from the cart.

Fixes #259 


## Type of change

Please give a X on it which is applicable

- (mentioned in the issue number)

# How Has This Been Tested?

Test A: Mandatory Size Selection:
- Navigate to the product page.
- Attempt to add an item to the cart without selecting a size.
- Verify that an error message or indication prevents the item from being added.
- Select a size and then add the item to the cart.
- Confirm that the item is successfully added to the cart.

Test B: Clear Cart After Checkout:
- Add multiple items to the cart.
- Proceed to the checkout page.
- Complete the checkout process.
- Verify that a confirmation message is displayed, and the cart is emptied.
- Navigate back to the cart page to ensure it is empty.

# Screenshots and Videos:

BUG 1 FIX
![Screenshot 2024-06-22 211743](https://github.com/pooranjoyb/popShop/assets/127378920/61354d4a-5017-4c43-8edd-54e0478b2b0f)

BUG 2 FIX
![image](https://github.com/pooranjoyb/popShop/assets/127378920/37d563cb-3dc5-4adb-b703-202308099106)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/cde29020-8d3e-4850-ac17-9bd924b5b12c)
![image](https://github.com/pooranjoyb/popShop/assets/127378920/15221c20-6a92-44aa-ac85-879840690c31)



# Checklist:
give a X on it which is applicable
